### ISSUE TITLE
Auto-suggest in link editor doesn't disappear

### ISSUE THREAD

**Issue by johngodley** [WordPress/gutenberg#12221]
It's not easy to dismiss the auto-suggest popup when editing a link, either with mouse or keyboard.

Here I am adding a link in a paragraph and type two characters. The auto-suggest appears:

<img width="390" alt="edit_post_ _wordpress_latest_ _wordpress" src="https://user-images.githubusercontent.com/1277682/48904647-f03e1a00-ee56-11e8-8339-0aa736e83bbd.png">

I don't want to pick anything from the list, but cannot dismiss it by clicking or tabbing onto another input. Here I have tried to open the link options but cannot access them as the suggestions won't go away:

<img width="400" alt="edit_post_ _wordpress_latest_ _wordpress" src="https://user-images.githubusercontent.com/1277682/48904707-1ebbf500-ee57-11e8-980f-7c68cc2a988c.png">

The only way to dismiss it is by closing the link editor.

**To Reproduce**
1. Create a paragraph
2. Add a link
3. Type a few characters to get some post suggestions
4. Try and dismiss the suggestions without closing the link editor

**Expected behavior**
The suggestions to close when focus is lost from the edit area. Also, hitting escape should close the list.

**Comment by timwright12** [comment]
@johngodley I think this may have gotten fixed at some point. I just testing in Chrome, Firefox, and Safari. Both ESC and blur (clicking outside the menu) worked to close the component. I will say that I was not able to [tab] out of the area, but that's an expected focus-trapping behavior for something like this.

cc @tofumatt

**Comment by afercia** [comment]
This is the expected behavior of an ARIA combobox with suggestions. It also matches the behavior of suggestions in the Classif block. Closing for now. Conversation can continue also on closed issues :)
